proto/shentu/gov/v1alpha1/gov.proto

```diff
@@ -11,89 +11,68 @@ import "google/protobuf/any.proto";
-    //  months.
-    google.protobuf.Duration max_deposit_period = 3 [
-        (gogoproto.nullable)    = false,
-        (gogoproto.stdduration) = true,
-        (gogoproto.jsontag)     = "max_deposit_period,omitempty",
-        (gogoproto.moretags)    = "yaml:\"max_deposit_period\""
-    ];
+  //  Maximum period for CTK holders to deposit on a proposal. Initial value: 2
+  //  months.
+  google.protobuf.Duration max_deposit_period = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.jsontag) = "max_deposit_period,omitempty",
+    (gogoproto.moretags) = "yaml:\"max_deposit_period\""
+  ];
 }
 
 // TallyParams defines the params for tallying votes on governance proposals.
 message TallyParams {
-    cosmos.gov.v1beta1.TallyParams default_tally = 1;
-    cosmos.gov.v1beta1.TallyParams certifier_update_security_vote_tally = 2;
-    cosmos.gov.v1beta1.TallyParams certifier_update_stake_vote_tally = 3;
+  cosmos.gov.v1beta1.TallyParams default_tally = 1;
+  cosmos.gov.v1beta1.TallyParams certifier_update_security_vote_tally = 2;
+  cosmos.gov.v1beta1.TallyParams certifier_update_stake_vote_tally = 3;
 }
-  
+
 message Proposal {
   option (gogoproto.equal) = true;
 
-  google.protobuf.Any content            = 1 [(cosmos_proto.accepts_interface) = "Content"];
-  uint64              proposal_id        = 2 [(gogoproto.jsontag) = "id", (gogoproto.moretags) = "yaml:\"id\""];
-  ProposalStatus      status             = 3 [(gogoproto.moretags) = "yaml:\"proposal_status\""];
-  bool is_proposer_council_member = 4 [ (gogoproto.moretags) = "yaml:\"is_proposer_council_member\"" ];
-  string proposer_address = 5 [ (gogoproto.moretags) = "yaml:\"proposer_address\"" ];
+  google.protobuf.Any content = 1 [(cosmos_proto.accepts_interface) = "Content"];
+  uint64              proposal_id = 2 [(gogoproto.jsontag) = "id", (gogoproto.moretags) = "yaml:\"id\""];
+  ProposalStatus      status = 3 [(gogoproto.moretags) = "yaml:\"proposal_status\""];
+  bool is_proposer_council_member = 4 [(gogoproto.moretags) = "yaml:\"is_proposer_council_member\""];
+  string proposer_address = 5 [(gogoproto.moretags) = "yaml:\"proposer_address\""];
   cosmos.gov.v1beta1.TallyResult         final_tally_result = 6
-      [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"final_tally_result\""];
+  [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"final_tally_result\""];
   google.protobuf.Timestamp submit_time = 7
-      [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"submit_time\""];
+  [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"submit_time\""];
   google.protobuf.Timestamp deposit_end_time = 8
-      [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"deposit_end_time\""];
+  [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"deposit_end_time\""];
```

Comment:
these should be indented for readability since it's with in the scope of the line above.

Reply:
Done